content-services/latest/config/mtls.md

```diff
@@ -0,0 +1,283 @@
+
+You can create the keystores, truststores and certificates required to configure SSL/mutual TLS authentication between different services in Content Services, such as the repository and Solr.
+
+The diagram shown is an overview of mTLS structure:
+
+![mTLS diagram]({% link content-services/images/mtls-diagram.png %})
+
+## Generate secure keys for SSL communication
+
+Use this information to generate certificates for SSL/mutual TLS authentication between the repository and Content Services, using secure keys specific to your installation.
+
+The old script version can still be used and its description is placed here [Alfresco Docs - Secure keys]({% link search-services/latest/config/keys.md %}), it can be replaced with executions of new scripts, that allow for more granularity and control (for example, excluding Solr).
+
+For both approaches run_additional script can be used to generate additional sets of keystores/truststores to be used by other services. So adding mTLS set up for Transform Services with mTLS already present for Solr is possible with run_additional scripts usage.
+
+A certificates generator script, `run.sh` (for Linux) and `run.cmd` (for Windows) is provided in a GitHub project. The script consists of two parts - the first part is based on OpenSSL (to generate the certificates), and the second part is based on the Java `keytool` (to build the keystores and truststores). Here, we'll focus on running the script for a standalone Linux or Windows operating system.
+
+Before you start, you must already have OpenSSL and `keytool` available in your system path.
+
+1. Browse to the [https://github.com/Alfresco/alfresco-ssl-generator](https://github.com/Alfresco/alfresco-ssl-generator) GitHub project and click **Clone** or **Download**.
+
+    If you downloaded the project, extract the files to a suitable location.
+
+2. Change directory to the following location and run the script:
+
+    (For Linux)
+
+    ```bash
+    cd /ssl-tool
+    ./run.sh
+    ```
+
+    (For Windows)
+
+    ```bash
+    cd ssl-tool-win
+    run.cmd
+    ```
+
+    By default, this creates a `keystores` folder in your current working directory.
+
+    > **Note:** If the `keystores` folder isn't empty, the script exits without producing any keystore or truststore. You can safely, remove the `keystores` folder if you need to re-run the script.
+
+See [Keystore directory structure]({% link search-services/latest/config/keys.md %}#keystore-directory-structure) for more and [Customize certificate generation]({% link search-services/latest/config/keys.md %}#customize-certificate-generation) for a full list of parameters that allow you to customize your certificates. It is recommended that you set your own passwords when generating certificates.
+
+## Customize certificate generation
+
+Here is a full list of parameters that allow you to customize your certificates. These parameters will override the default values listed in the `run.sh` and `run.cmd` scripts.

```

Comment:
I'd add a header here, similar to "New scripts summary" that says "Existing script summary" "Old script summary" or "Legacy script summary".

"run.sh" and "run.cmd" are old versions, non-configurable though still viable for usage, it's not like they've become deprecated, just a lot more configurable options have appeared. I don't know how to exactly express that sentiment though.

Reply:
I've updated the documentation.